src/Items/Calendar/CalendarView.tsx

```diff
@@ -4,16 +4,22 @@ import { LoginRequired } from "../../Auth/Login/helpers/LoginRequired";
 import { ItemDrawer } from "../Drawer/ItemDrawer";
 import { useItems } from "../state/ItemContext";
 import { ItemDrawerProvider } from "../state/ItemDrawerContext";
+import { ItemType, Status } from "../types";
 import { Calendar } from "./Calendar";
 
+const defaultItem = {
```

Comment:
sounds like a template rather than concrete item

Reply:
It's Partial<Item>, I should have added the type :+1: